charts/nopo11y/templates/defaultSLOs.yaml

```diff
@@ -165,11 +169,11 @@ spec:
         dashboard: {{ .Values.grafanaURL }}/d/slo-detail?var-service={{ include "app.label" . }}
         {{- end }}
         summary: SLO to measure response time - {{ .Values.latencySLO }}% of the time requests should
-          be succesfully served in < 1s. When you receive this alert it means that
+          be succesfully served in < {{ .Values.latency }}s. When you receive this alert it means that
```

Comment:
You need to change it to `ms` from `s`, check for others as well.

Reply:
Changes s to ms for latency in latency SLO using nginx ingress metrics